docs/guides/migrating-from-cawemo.md

```diff
@@ -0,0 +1,57 @@
+---
+id: migrating-from-cawemo
+title: Migrating Diagrams from Cawemo
+description: "Migrating Diagrams from Cawemo to Camunda Platform 8 Web Modeler"
+---
+
+## Migrating Diagrams from Cawemo to Camunda Platform 8 Web Modeler
+
+This guide steps through how to migrate BPMN diagrams created in Cawemo to Camunda Platform 8 Web Modeler.
+
+- You should consider migrating diagrams from Cawemo to Camunda Platform 8 Web Modeler if:
+- You want to continue working on your diagrams on the latest Camunda Platform.
+- You are exploring automating your processes.
+- You are in need of certain features that can only be found in Camunda Platform 8 (e.g. [BPMN message buffering](https://docs.camunda.io/docs/components/concepts/messages/#message-buffering), big [multi-instance constructs](https://docs.camunda.io/docs/components/modeler/bpmn/multi-instance/), the new Connectors framework, or the improved collaboration features in web modeler).
+
+## Prerequisites
+
+- Ensure you have a valid [Camunda Platform 8 account](https://docs.camunda.io/docs/guides/getting-started/), or sign up if you still need one.
+- A BPMN diagram in your Cawemo account.
+
+## Migrate your BPMN diagram
+
+Take the following steps to migrate your BPMN diagram in Cawemo to Camunda Platform 8 [Web Modeler](https://docs.camunda.io/docs/components/modeler/web-modeler/launch-cloud-modeler/):
+
+1. Log in to Cawemo.
+2. Click the **Project** folder on the **Home** page of the BPMN diagram you want to transfer.
+3. Click on the BPMN diagram you want to transfer.
+
+:::note
+If the BPMN file has been uploaded to Cawemo from Desktop Modeler for execution on Camunda Platform 7, the file cannot be uploaded to Web Modeler.
+:::
+
+4. Click **Download** or **export**
+5. Click **Download as BPMN 2.0 XML**.
+6. Log in to Camunda Platform 8.
+7. Click **Modeler** to open Web Modeler.
+8. Open the project you want to transfer your BPMN file to or create a new project by clicking **New project**.
+9. Click **New** > **Upload files**.
+10. Open the BPMN XML file downloaded from Cawemo.
+
+:::note
+While unsupported elements are imported in Web Modeler, they are not supported for deployment and will be highlighted in the error panel.
+:::
+
+## Additional resources
+
+Not all elements in Cawemo are supported in Camunda Platform 8 Web Modeler. Find details on BPMN coverage in Camunda Platform 8 in the [BPMN coverage documentation](https://docs.camunda.io/docs/components/modeler/bpmn/bpmn-coverage/). We are investing in supporting the elements currently available on Camunda Platform 7.
+
+## Next step
+
+Learn more about [migrating from Camunda 7 to Camunda 8](https://docs.camunda.io/docs/guides/migrating-from-camunda-platform-7/).
+
+## FAQ
+
+Questions: Why do I get the following error The following 1 file is invalid and can't be uploaded: “" when uploading my BPMN file?
```

Comment:
```suggestion
Questions: Why do I get the error `The following 1 file is invalid and can't be uploaded: “"` when uploading my BPMN file?
```

1. I'd remove the word "following" (the first usage) because it seems unnecessary
2. I can't find anything in the style guide or cheatsheet about how we handle inline error messages, but I'd expect it to stand out from the rest of the sentence somehow. Maybe @akeller knows what the standard is.

(Apparently I didn't save this comment before I submitted my previous review 😅 )